python/tvm/relay/op/_transform.py

```diff
@@ -687,7 +678,6 @@ def argwhere_shape_func(attrs, inputs, out_ndims):
 
 
 _reg.register_shape_func("scatter", False, elemwise_shape_func)
```

Comment:
Do you plan to deprecate `relay.scatter` too?

Reply:
https://github.com/apache/tvm/pull/14019